java-client/src/main/java/org/apache/pegasus/client/PegasusScanner.java

```diff
@@ -188,6 +193,16 @@ private void asyncStartScan() {
         new Table.ClientOPCallback() {
           @Override
           public void onCompletion(client_operator clientOP) throws Throwable {
+            if (!(clientOP instanceof rrdb_get_scanner_operator)) {
+              logger.error(
+                  "scan rpc callback, encounter logic error, we just abandon this scan, "
+                      + "tableName({}), appId({})",
+                  _table.getTableName(),
```

Comment:
These are all C style naming? Can we remove the leading '_'?

Reply:
Yes, but in this project, many of the formats are C-grammatical formats, and the current way of writing this is entirely because the attribute names are like this. I want to handle them uniformly in subsequent PRs.